app/views/inventories/index.html.erb

```diff
@@ -1,48 +1,46 @@
-<div class="card">
-  <div class="tab">
-    <button class="tablinks active" onclick="openSection(event, 'inventories')">Equipos</button>
-    <button class="tablinks" onclick="openSection(event, 'supplies')">Insumos</button>
-  </div>  
-  <div id="inventories" class="tabcontent default" style="display: block;">
-    <section class="inventories-index">
-      <div class="card">
-        <div class="content">
-          <div class="header card-header-text sical-card-header">
-          <h1 class="sical-card-title">
-            Equipos
-          </h1>      
-            <div>
-              <%= link_to 'Nuevo Item', new_inventory_path, class: "btn btn-primary align-right" %>
-            </div>
+<div class="tab">
+  <button class="tablinks active" onclick="openSection(event, 'inventories')">Equipos</button>
```

Comment:
Don't use inline functions of js. The function should be on the corresponding file.